### ISSUE TITLE
Animal Populations Decrease over Time

### ISSUE THREAD

**Issue by WeaselDog** [StrangeLoopGames/EcoIssues#15822]
All of the animal populations decrease over time, regardless of initial populations. There is an initial dip after a day then the populations continue to decrease slowly over time. This can be tested using the /ff command.

I have tested with SpreadRate & SpeciesChangeRate using very high numbers and it does not help this problem.

**Comment by WeaselDog** [comment]
Lowering in priority because it is solved for most animals. We just need to figure out a few of the predators.

**Comment by SlayksWood** [comment]
yea fox, coyote and wolf. And tarantula:
![image](https://user-images.githubusercontent.com/45708377/79845685-1d815580-83c6-11ea-8b29-46701268ed04.png)

**Comment by WeaselDog** [comment]
Blocked by #16415

**Comment by SlayksWood** [comment]
fixed descending graph, but we lost population increase at game beginning add to another issue:
https://github.com/StrangeLoopGames/EcoIssues/issues/15896#issuecomment-643924819
